createwindow/createwindow.go

```diff
@@ -152,6 +153,23 @@ func SetupChrome(chromeWindow ChromeWindow, URL string) error {
 	// TODO: use cookies to login if user allows.
 	return nil
 }
+func DisableCrashedBubble(s string) error {
+	fmt.Println(s)
+	_, err := os.Stat(s)
+	if os.IsNotExist(err) {
+		fmt.Println("preferences file does not exist, no changes to be made")
+		return nil
+	}
+	read, err := ioutil.ReadFile(s)
+	if err != nil {
+		return err
+	}
+	newContents := strings.Replace(string(read), "\"exit_type\":\"Crashed\"", "\"exit_type\":\"Normal\"", -1)
+	if err = ioutil.WriteFile(s, []byte(newContents), 0644); err != nil {
```

Comment:
rather than hardcoding 0644, it might be better to read the existing mode of the file (from your stat call above)